### DISCUSSION TITLE
Gesture or button for next/previous article

### DISCUSSION THREAD

**Discussion by etern** [jocmp/capyreader#241] [Feature Requests]
While reading article, swipe left or right to switch to the previous or next article

**Comment by jocmp** [comment]
@etern good idea. I definitely want to add this sometime soon.

**Comment by jocmp** [comment]
Haven't started this yet, but it's planned. I'm tracking it via https://github.com/jocmp/capyreader/issues/265

**Comment by jocmp** [comment]
This was added in [2024.09.1051](https://github.com/jocmp/capyreader/releases/tag/2024.09.1051). Closing for now.
